api.go

```diff
@@ -548,12 +560,13 @@ func EnterContainer(podID, containerID string, cmd Cmd) (*Pod, *Container, *Proc
 	}
 
 	// Enter it.
-	process, err := c.enter(cmd)
+	cImpl := c.(*Container)
```

Comment:
Not needed if we agree that `fetchContainer()` should not return `VCContainer` type.